lib/const.rb

```diff
@@ -50,7 +50,7 @@
 ARCH_LIB        = "lib#{CREW_LIB_SUFFIX}"
 
 # Glibc version can be found from the output of libc.so.6
-LIBC_VERSION = Etc.confstr(Etc::CS_GNU_LIBC_VERSION).split.last
+LIBC_VERSION = ENV.fetch('LIBC_VERSION', Etc.confstr(Etc::CS_GNU_LIBC_VERSION).split.last )
```

Comment:
What was the problem with the old method?

Reply:
Trying to set `LIBC_VERSION` manually for when we need to override it, as when we are building the various packages with that used in the version field.